python/paddle/fluid/dygraph/dygraph_to_static/call_transformer.py

```diff
@@ -36,8 +36,8 @@ def _is_builtin_call(self, node):
         assert isinstance(node, gast.Call)
         func_str = ast_to_source_code(node.func).strip()
         try:
-            from paddle.fluid.dygraph.dygraph_to_static.convert_call_func import is_builtin
-            return eval("is_builtin({})".format(func_str))
+            from paddle.fluid.dygraph.dygraph_to_static.convert_call_func import is_builtin_call
+            return eval("is_builtin_call({})".format(func_str))
```

Comment:
Here, utility function named "is_builtin_call" doesn't mean "is_builtin_call", which requires callers to read more code of "is_builtin_call" to know the details. I would suggest to put the logic of "not is_builtin_len" at where you use it to increase readability.

Reply:
Thanks，done.